addon/notifications/service.js

```diff
@@ -44,5 +44,19 @@ export default Service.extend({
         .then(() => resolve(notifications))
         .catch((e) => reject(e));
     });
+  },
+
+  fetchFeed(field) {
+    return this.ajax.request(this.fetchFeedUrl(field), {
```

Comment:
Why not using native `fetch` of js?

Reply:
keeping it consistent with the rest of the file. Will probably be updated to TS anyway and we'll use fetch then.

Reply:
Ok :+1:

Reply:
i think it's because it's an existing service and the other calls are done this way